### DISCUSSION TITLE
Is it possible to bind Lidl Zigbee products?

### DISCUSSION THREAD

**Discussion by utigernils** [Koenkk/zigbee2mqtt#6963] [Devices]
Hey guys, I can't figure out how to bind my Lidl ZigBee lights to my Lidl ZigBee switch in zigbee2mqtt so I can use zigbee2mqtt and the switch to control my lights.

I'm new to this stuff
I would appreciate help.

**Comment by danieledwardgeorgehitchcock** [comment]
Hi,

Your best bet would be to follow these comprehensive tutorials:

[Binding](https://www.zigbee2mqtt.io/information/binding.html)
[Groups](https://www.zigbee2mqtt.io/information/groups.html)
